scalereg/scale_templates/reg6/reg_kiosk_clear.html

```diff
@@ -0,0 +1,42 @@
+<html>
```

Comment:
@leizleiz Wait are you sure for TXLF we always run on head of master, and use the kiosk pages every year.